plugin/mlsdbonlinelocator.cpp

```diff
@@ -80,6 +82,8 @@ MlsdbOnlineLocator::MlsdbOnlineLocator(QObject *parent)
     QSettings settings(MLSConfigFile, QSettings::IniFormat);
     m_fallbacksLacf = settings.value("MLS/FALLBACKS_LACF", true).toBool();
     m_fallbacksIpf = settings.value("MLS/FALLBACKS_IPF", true).toBool();
+    m_mlsKey = settings.value("MLS/CUSTOM_KEY", "").toString();
```

Comment:
The default value can be also skipped if it's an empty string. 

Besides that QString() should be usually better than "" literals. That also for the couple cases below.

Reply:
Changed